coral/media/js/views/components/plugins/licensing-workflow.js

```diff
@@ -412,26 +413,54 @@ define([
           ]
         },
         {
-          title: 'Cover Letter',
-          name: 'cover-letter-step',
+          name: "ea74e8cb-ce03-49c6-aeef-b5a0e62f8cdf",
+          title: "Letter",
           required: false,
           layoutSections: [
             {
               componentConfigs: [
                 {
-                  componentName: 'license-cover-letter',
-                  uniqueInstanceName: 'cover-letter',
-                  tilesManaged: 'one',
                   parameters: {
-                    graphid: 'cc5da227-24e7-4088-bb83-a564c4331efd',
-                    nodegroupid: '0dcf7c74-53d5-11ee-844f-0242ac130008',
-                    resourceid: "['init-step']['app-id'][0]['resourceid']['resourceInstanceId']"
-                  }
+                    graphid: "cc5da227-24e7-4088-bb83-a564c4331efd",
+                    resourceid: "['init-step']['app-id'][0]['resourceid']['resourceInstanceId']",
+                    // hiddenNodes: ["49c65ece-f5af-11ee-9f07-0242ac170006"],
+                    nodegroupid: "87bdb8d8-3927-11ef-ab34-0242ac140006",
+                    semanticName: "Correspondence",
+                    letterMetatype: "976e49fe-3928-11ef-ab34-0242ac140006",
+                    letterTypeNode: "56364572-3928-11ef-b242-0242ac140006",
+                    letterResourceNode: "21319570-3928-11ef-b242-0242ac140006"
+                  },
+                  noTileSidebar: true,
+                  tilesManaged: "many",
+                  componentName: "file-template",
+                  uniqueInstanceName: "letter-template"
                 }
               ]
             }
-          ]
+          ],
+          workflowstepclass: "workflow-form-component"
         },
+        // {
```

Comment:
Can commented code that won't be used removed

Reply:
removed